### PR TITLE
Use BEM class names

### PR DESCRIPTION
**Pull request by san00**

### CODE REVIEW

File: dist/css/style.css [hunk 1]
```diff
@@ -16,39 +16,39 @@ html {
   -webkit-box-sizing: inherit;
   box-sizing: inherit; }
 
-.info {
+.about {
+  margin-left: 1rem;
+  margin-right: 1rem; }
+
+.about_title {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__title (two underscores)

File: dist/css/style.css [hunk 2]
```diff
@@ -16,39 +16,39 @@ html {
...
   font-size: 1.1em;
   margin-bottom: calc(1rem * 2);
   width: 3.5rem; }
 
-.about {
-  margin-left: 1rem;
-  margin-right: 1rem; }
-
-.about--info {
+.about_header {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 3]
```diff
@@ -16,39 +16,39 @@ html {
...
   color: #262626;
   font-size: 3.2em;
   font-weight: bold;
   line-height: 1.11em;
   margin-bottom: 1rem; }
 
-.about--links {
+.about_info {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 4]
```diff
@@ -16,39 +16,39 @@ html {
...
   color: #3f3f3f;
   font-size: 1.6em;
   line-height: 1.5em;
   margin-bottom: calc(1rem * 2); }
 
-.links {
+.about_links {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 5]
```diff
@@ -16,39 +16,39 @@ html {
...
   color: #cf2691;
   -webkit-transition: background-color 0.2s;
   transition: background-color 0.2s; }
 
-.links:hover {
+.about_links:hover {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 6]
```diff
@@ -16,39 +16,39 @@ html {
...
   background-color: gold; }
 
-.aside-img {
+.aside_img {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 7]
```diff
@@ -59,35 +59,33 @@ html {
 .icons {
   visibility: hidden; }
 
-.github,
-.twitter {
+.icons_github,
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 8]
```diff
@@ -59,35 +59,33 @@ html {
...
+.icons_twitter {
   height: 2.25em;
   visibility: hidden;
   width: 2.25em; }
 
-.twitter {
+.icons_twitter {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 9]
```diff
@@ -59,35 +59,33 @@ html {
...
   fill: #1da1f2; }
 
-.twitter:hover {
+.icons_twitter:hover {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 10]
```diff
@@ -59,35 +59,33 @@ html {
...
   fill: lightblue; }
 
-.github:hover {
+.icons_github:hover {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 11]
```diff
@@ -59,35 +59,33 @@ html {
...
   fill: darkgrey; }
 
 @media screen and (min-width: 26.1rem) and (max-width: 46rem) {
-  .aside-img {
-    margin-bottom: calc(1rem * 4); }
-  .info {
+  .aside_img {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 12]
```diff
@@ -59,35 +59,33 @@ html {
...
     margin-bottom: calc(1rem * 4); }
-  .about--info {
+  .about_header {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 13]
```diff
@@ -59,35 +59,33 @@ html {
...
     margin-bottom: 1rem; }
-  .about--links {
+  .about_info {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 14]
```diff
@@ -59,35 +59,33 @@ html {
...
     margin-bottom: calc(1rem * 2); }
   .icons,
-  .github,
-  .twitter {
+  .icons_github,
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 15]
```diff
@@ -59,35 +59,33 @@ html {
...
+  .icons_twitter {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 16]
```diff
@@ -108,9 +106,9 @@ html {
     -ms-flex: 2;
     flex: 2;
     margin: auto 2rem 7rem 5rem; }
-  .about--info {
+  .about_header {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 17]
```diff
@@ -108,9 +106,9 @@ html {
...
     margin-bottom: 1rem; }
-  .about--links {
+  .about_info {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 18]
```diff
@@ -121,16 +119,16 @@ html {
     justify-content: center;
     margin-right: calc(1rem * 4); }
   .icons,
-  .github,
-  .twitter {
+  .icons_github,
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 19]
```diff
@@ -121,16 +119,16 @@ html {
...
+  .icons_twitter {
     visibility: visible; }
-  .github {
+  .icons_github {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 20]
```diff
@@ -121,16 +119,16 @@ html {
...
     margin-left: 1rem; }
   .aside {
     -webkit-box-flex: 1;
     -ms-flex: 1;
     flex: 1; }
-  .aside-img {
+  .aside_img {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 21]
```diff
@@ -148,10 +146,10 @@ html {
     -webkit-box-pack: center;
     -ms-flex-pack: center;
     justify-content: center; }
-  .github {
+  .icons_github {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 22]
```diff
@@ -148,10 +146,10 @@ html {
...
     margin-left: 1rem; }
-  .about--info,
-  .about--links {
+  .about_header,
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/css/style.css [hunk 23]
```diff
@@ -148,10 +146,10 @@ html {
...
+  .about_info {
```
Comment (IgnaciodeNuevo) [inline dist/css/style.css]: Change to .about__... (two underscores)

File: dist/index.html [hunk 24]
```diff
@@ -13,19 +13,19 @@
 <body>
   <div class="content">
     <aside class="aside">
-      <img class="aside-img" src="images/jeremy-galliani-697655-unsplash.jpg" alt="mural on wall">
+      <img class="aside_img" src="images/jeremy-galliani-697655-unsplash.jpg" alt="mural on wall">
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as above

File: dist/index.html [hunk 25]
```diff
@@ -13,19 +13,19 @@
...
     </aside>
     <section class="about">
-      <p class="info">About</p>
-      <p class="about--info">Osania is a front end developer.</p>
-      <p class="about--links">I make things with HTML, CSS and JavaScript. <br> See my projects on <a class="links"
+      <p class="about_title">About</p>
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as above

File: dist/index.html [hunk 26]
```diff
@@ -13,19 +13,19 @@
...
+      <p class="about_header">Osania is a front end developer.</p>
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as above

File: dist/index.html [hunk 27]
```diff
@@ -13,19 +13,19 @@
...
+      <p class="about_info">I make things with HTML, CSS and JavaScript. <br> See my projects on <a class="about_links"
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as above

File: dist/index.html [hunk 28]
```diff
@@ -13,19 +13,19 @@
...
           href="https://github.com/san00">github</a>,
-        follow me on <a class="links" href="https://twitter.com/sancodes">twitter</a> or say hello via <a class="links"
+        follow me on <a class="about_links" href="https://twitter.com/sancodes">twitter</a> or say hello via <a class="about_links"
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as above

File: dist/index.html [hunk 29]
```diff
@@ -13,19 +13,19 @@
...
           href="mailto:76d5xcxspvjm@opayq.com">email</a>.</p>
         <div class="icons">
-          <a href="https://twitter.com/sancodes" class="link-text"><svg class="twitter" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+          <a href="https://twitter.com/sancodes" class="link-text"><svg class="icons_twitter" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as above

File: dist/index.html [hunk 30]
```diff
@@ -13,19 +13,19 @@
...
               <path d="M24 4.557c-.883.392-1.832.656-2.828.775 1.017-.609 1.798-1.574 2.165-2.724-.951.564-2.005.974-3.127 1.195-.897-.957-2.178-1.555-3.594-1.555-3.179 0-5.515 2.966-4.797 6.045-4.091-.205-7.719-2.165-10.148-5.144-1.29 2.213-.669 5.108 1.523 6.574-.806-.026-1.566-.247-2.229-.616-.054 2.281 1.581 4.415 3.949 4.89-.693.188-1.452.232-2.224.084.626 1.956 2.444 3.379 4.6 3.419-2.07 1.623-4.678 2.348-7.29 2.04 2.179 1.397 4.768 2.212 7.548 2.212 9.142 0 14.307-7.721 13.995-14.646.962-.695 1.797-1.562 2.457-2.549z" /></svg>Twitter</a>
-          <a href="https://github.com/san00"><svg class="github" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+          <a href="https://github.com/san00"><svg class="icons_github" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```
Comment (IgnaciodeNuevo) [inline dist/index.html]: Same as above

File: src/index.html [hunk 31]
```diff
@@ -13,19 +13,19 @@
 <body>
   <div class="content">
     <aside class="aside">
-      <img class="aside-img" src="images/jeremy-galliani-697655-unsplash.jpg" alt="mural on wall">
+      <img class="aside_img" src="images/jeremy-galliani-697655-unsplash.jpg" alt="mural on wall">
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as above

File: src/index.html [hunk 32]
```diff
@@ -13,19 +13,19 @@
...
     </aside>
     <section class="about">
-      <p class="info">About</p>
-      <p class="about--info">Osania is a front end developer.</p>
-      <p class="about--links">I make things with HTML, CSS and JavaScript. <br> See my projects on <a class="links"
+      <p class="about_title">About</p>
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as above

File: src/index.html [hunk 33]
```diff
@@ -13,19 +13,19 @@
...
+      <p class="about_header">Osania is a front end developer.</p>
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as above

File: src/index.html [hunk 34]
```diff
@@ -13,19 +13,19 @@
...
+      <p class="about_info">I make things with HTML, CSS and JavaScript. <br> See my projects on <a class="about_links"
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as above

File: src/index.html [hunk 35]
```diff
@@ -13,19 +13,19 @@
...
           href="https://github.com/san00">github</a>,
-        follow me on <a class="links" href="https://twitter.com/sancodes">twitter</a> or say hello via <a class="links"
+        follow me on <a class="about_links" href="https://twitter.com/sancodes">twitter</a> or say hello via <a class="about_links"
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as above

File: src/index.html [hunk 36]
```diff
@@ -13,19 +13,19 @@
...
           href="mailto:76d5xcxspvjm@opayq.com">email</a>.</p>
         <div class="icons">
-          <a href="https://twitter.com/sancodes" class="link-text"><svg class="twitter" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+          <a href="https://twitter.com/sancodes" class="link-text"><svg class="icons_twitter" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as above

File: src/index.html [hunk 37]
```diff
@@ -13,19 +13,19 @@
...
               <path d="M24 4.557c-.883.392-1.832.656-2.828.775 1.017-.609 1.798-1.574 2.165-2.724-.951.564-2.005.974-3.127 1.195-.897-.957-2.178-1.555-3.594-1.555-3.179 0-5.515 2.966-4.797 6.045-4.091-.205-7.719-2.165-10.148-5.144-1.29 2.213-.669 5.108 1.523 6.574-.806-.026-1.566-.247-2.229-.616-.054 2.281 1.581 4.415 3.949 4.89-.693.188-1.452.232-2.224.084.626 1.956 2.444 3.379 4.6 3.419-2.07 1.623-4.678 2.348-7.29 2.04 2.179 1.397 4.768 2.212 7.548 2.212 9.142 0 14.307-7.721 13.995-14.646.962-.695 1.797-1.562 2.457-2.549z" /></svg>Twitter</a>
```
Comment (IgnaciodeNuevo) [inline src/index.html:27]: Same as above

File: src/index.html [hunk 38]
```diff
@@ -13,19 +13,19 @@
...
-          <a href="https://github.com/san00"><svg class="github" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+          <a href="https://github.com/san00"><svg class="icons_github" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```
Comment (IgnaciodeNuevo) [inline src/index.html]: Same as above

File: src/sass/style.scss [hunk 39]
```diff
@@ -3,7 +3,7 @@ $primary-text-colour: #262626;
 $secondary-text-colour: #3f3f3f;
 $accent-border-colour: #f4b69c;
 $hover-highlight-colour: gold;
-$info-links-colour: #cf2691;
+$about_links-colour: #cf2691;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Don't use BEM in Sass variables

File: src/sass/style.scss [hunk 40]
```diff
@@ -35,88 +35,95 @@ html {
     box-sizing: inherit;
 }
 
-.info {
+.about {
+    margin-left: $spacing-m;
+    margin-right: $spacing-m;
+}
+
+.about_title {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 41]
```diff
@@ -35,88 +35,95 @@ html {
...
+.about_header {
     color: $primary-text-colour;
     font-size: 3.2em;
     font-weight: bold;
     line-height: 1.11em;
     margin-bottom: $spacing-m;
 }
 
-.about--links {
+.about_info {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Same as above

File: src/sass/style.scss [hunk 42]
```diff
@@ -35,88 +35,95 @@ html {
...
     color: $secondary-text-colour;
     font-size: 1.6em;
     line-height: 1.5em;
     margin-bottom: $spacing-l;
 }
 
-.links {
-    color: $info-links-colour;
+.about_links {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Same as above

File: src/sass/style.scss [hunk 43]
```diff
@@ -35,88 +35,95 @@ html {
...
+    color: $about_links-colour;
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Don't use unserscore in sass variables (BEM is to name classes not variables)

File: src/sass/style.scss [hunk 44]
```diff
@@ -35,88 +35,95 @@ html {
...
     transition: background-color 0.2s;
 }
 
-.links:hover {
+.about_links:hover {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 45]
```diff
@@ -35,88 +35,95 @@ html {
...
     background-color: $hover-highlight-colour;
 }
 
-.aside-img {
+.aside_img {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 46]
```diff
@@ -35,88 +35,95 @@ html {
...
     width: 100vw;
 }
+
 .icons {
     visibility: hidden;
 }
-.github,
-.twitter {
+
+.icons_github,
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 47]
```diff
@@ -35,88 +35,95 @@ html {
...
+.icons_twitter {
     height: 2.25em;
     visibility: hidden;
     width: 2.25em;
 }
-.twitter {
+
+.icons_twitter {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 48]
```diff
@@ -35,88 +35,95 @@ html {
...
     fill: #1da1f2;
 }
-.twitter:hover {
+
+.icons_twitter:hover {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 49]
```diff
@@ -35,88 +35,95 @@ html {
...
     fill: lightblue;
 }
-.github:hover {
+
+.icons_github:hover {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 50]
```diff
@@ -35,88 +35,95 @@ html {
...
     fill: darkgrey;
 }
+
 // Adjusts layout between mobile & tablet screen sizes
 @media screen and (min-width: 26.1rem) and (max-width: 46rem) {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: - Use two unserscores becacause is an Element (in BEM).
- Wired media query, right? 26.1rem? Thing about what I told you :)

File: src/sass/style.scss [hunk 51]
```diff
@@ -35,88 +35,95 @@ html {
...
-    .aside-img {
-        margin-bottom: $spacing-xl;
-    }
-    .info {
+    .aside_img {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 52]
```diff
@@ -35,88 +35,95 @@ html {
...
         margin-bottom: $spacing-xl;
     }
-    .about--info {
+
+    .about_header {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 53]
```diff
@@ -35,88 +35,95 @@ html {
...
         margin-bottom: $spacing-m;
     }
-    .about--links {
+
+    .about_info {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 54]
```diff
@@ -35,88 +35,95 @@ html {
...
         margin-bottom: $spacing-l;
     }
+
     .icons,
-    .github,
-    .twitter {
+    .icons_github,
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 55]
```diff
@@ -147,32 +154,35 @@ html {
         margin: auto 2rem 7rem 5rem;
     }
 
-    .about--info {
+    .about_header {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 56]
```diff
@@ -35,88 +35,95 @@ html {
...
     .icons,
-    .github,
-    .twitter {
+    .icons_github,
+    .icons_twitter {
         visibility: visible;
     }
-    .github {
+
+    .icons_github {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 57]
```diff
@@ -147,32 +154,35 @@ html {
         margin: auto 2rem 7rem 5rem;
     }
 
-    .about--info {
+    .about_header {
         margin-bottom: $spacing-m;
     }
-    .about--links {
+
+    .about_info {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 58]
```diff
@@ -147,32 +154,35 @@ html {
...
     .icons {
         display: flex;
         justify-content: center;
         margin-right: $spacing-xl;
     }
+
     .icons,
-    .github,
-    .twitter {
+    .icons_github,
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 59]
```diff
@@ -147,32 +154,35 @@ html {
...
+    .icons_twitter {
         visibility: visible;
     }
 
-    .github {
+    .icons_github {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 60]
```diff
@@ -147,32 +154,35 @@ html {
...
         display: flex;
         justify-content: center;
         margin-right: $spacing-xl;
     }
+
     .icons,
-    .github,
-    .twitter {
+    .icons_github,
+    .icons_twitter {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 61]
```diff
@@ -147,32 +154,35 @@ html {
...
+    .icons_github {
         margin-left: $spacing-m;
     }
 
     .aside {
         flex: 1;
     }
 
-    .aside-img {
+    .aside_img {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 62]
```diff
@@ -185,15 +195,18 @@ html {
...
         }
-        .github {
+
+        .icons_github {
             margin-left: $spacing-m;
         }
-        .about--info,
-        .about--links {
+
+        .about_header,
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

File: src/sass/style.scss [hunk 63]
```diff
@@ -185,15 +195,18 @@ html {
...
             font-size: large;
         }
+
         .icons {
             display: flex;
             justify-content: center;
         }
-        .github {
+
+        .icons_github {
```
Comment (IgnaciodeNuevo) [inline src/sass/style.scss]: Use two unserscores becacause is an Element (in BEM).

**Review by IgnaciodeNuevo — CHANGES_REQUESTED** [review]
Review BEM documentation: http://getbem.com/naming/
Change to real BEM please :)

File: src/sass/style.scss (hunk 50 above)
Comment (san00) [reply src/sass/style.scss]: Re: Media query, should I be thinking more about device orientation rather than width here?

Comment (IgnaciodeNuevo) [reply src/sass/style.scss]: The problem is that you should not use REMS in MQ and the decimal value (26.1 is kind of wired for a mq)

Comment (san00) [reply src/sass/style.scss]: Mq updated

**Review by IgnaciodeNuevo — APPROVED** [review]

**Pull request merged**